ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -386,7 +373,6 @@ private[dao] final class TransactionsReader(
         )
       }
       .mapConcat(TransactionConversions.toGetActiveContractsResponse(_)(contextualizedErrorLogger))
-      .buffer(outputStreamBufferSize, OverflowStrategy.backpressure)
       .wireTap(response => {
         Spans.addEventToSpan(
```

Comment:
Idea: I am wondering if we could move this also up to keep core stream logic separated from logging/metrics

Reply:
Good idea. I'll do it in a separate PR in order to keep this one contained.